<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QMenu" name="menuFont_size">
     <property name="title">
      <string>Font size</string>
     </property>
     <addaction name="action12_pt"/>
     <addaction name="action18_pt"/>
     <addaction name="action24_pt"/>
    </widget>
    <addaction name="menuFont_size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action12ptx">
   <property name="text">
    <string>12ptx</string>
   </property>
  </action>
  <action name="action14ptx">
   <property name="text">
    <string>14ptx</string>
   </property>
  </action>
  <action name="action12_pt">
   <property name="text">
    <string>12 pt</string>
   </property>
  </action>
  <action name="action18_pt">
   <property name="text">
    <string>18 pt</string>
   </property>
  </action>
  <action name="action24_pt">
   <property name="text">
    <string>24 pt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
